privacyidea/lib/tokens/webauthn.py

```diff
@@ -1523,14 +1520,11 @@ def __init__(self,
         self.challenge = challenge
         self.origin = origin
         self.allow_credentials = allow_credentials
-        self.uv_required = uv_required
+        self.uv_required = uv_required or USER_VERIFICATION_LEVEL.PREFERRED
```

Comment:
what function? there is not function with a parameter of `USER_VERIFICATION_LEVEL`, everything is an options dict ;)

Reply:
I meant the `__init__()` constructor for the `WebAuthnAssertionResponse`.
There the `uv_required` is set to `False` by default and in the docstring described as type `bool`.
But here we set it to an entry of `USER_VERIFICATION_LEVEL` (or pass it to the constructor if `USER_VERIFICATION_REQUIREMENT` is in the parameters).
It behaves differently than the `WebAuthnRegistrationResponse`: https://github.com/privacyidea/privacyidea/blob/f49a5949abe1a399f229b698af14887f5b3f861f/privacyidea/lib/tokens/webauthntoken.py#L903

Reply:
i have updated the doc. having a `bool` there is stupid, because we have the `USER_VERIFICATION_LEVEL` enum, so we should use it. however, the goal of this pr is to fix the issue while leaving the behavior exactly the same and not refactoring the webauthn token.